### PR TITLE
OCT-1373: Timeline: make entries linkable

### PR DESCRIPTION
**Pull request by jmikolajczyk**

### CODE REVIEW

File: client/src/components/Proposals/ProposalsTimelineWidgetItem/ProposalsTimelineWidgetItem.tsx [hunk 1]
```diff
@@ -15,17 +15,39 @@ const ProposalsTimelineWidgetItem: FC<ProposalsTimelineWidgetItemProps> = ({
...
+        setInitialClientX(e.clientX);
+      }}
+      onMouseUp={e => {
+        if (!href) {return;}
+        if (initialClientX === e.clientX) {
+          window.open(href, '_blank');
+        }
+
+        setInitialClientX(null);
+      }}
```
Comment (aziolek) [inline client/src/components/Proposals/ProposalsTimelineWidgetItem/ProposalsTimelineWidgetItem.tsx:45]: Why not just `onClick`?

Comment (jmikolajczyk) [reply client/src/components/Proposals/ProposalsTimelineWidgetItem/ProposalsTimelineWidgetItem.tsx]: Done

File: client/cypress/e2e/proposals.cy.ts [hunk 2]
```diff
@@ -139,6 +140,38 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight }) =>
...
+    it('ProposalsTimelineWidgetItem with href opens link when clicked without mouse movement', () => {
+      const milestones = getMilestones();
+      cy.get('[data-test=ProposalsTimelineWidget]').should('be.visible');
+      cy.get('[data-test=ProposalsTimelineWidgetItem]').should('have.length', milestones.length);
+      for (let i = 0; i < milestones.length; i++) {
+        if (milestones[i].href) {
+          cy.get('[data-test=ProposalsTimelineWidgetItem]')
+            .eq(i)
+            .within(() => {
+              cy.get('[data-test=ProposalsTimelineWidgetItem__Svg--arrowTopRight]').should('exist');
```
Comment (aziolek) [inline client/cypress/e2e/proposals.cy.ts:153]: Shouldn't it be `be.visible`? Exist doesn't mean the element is actually visible to the user.
```
cy.get('[data-test=ProposalsTimelineWidgetItem__Svg--arrowTopRight]').should('be.visible');
```

Comment (jmikolajczyk) [reply client/cypress/e2e/proposals.cy.ts]: Changed

**Review by aziolek — APPROVED** [review]

**Pull request merged**